app/scripts/metamask-controller.js

```diff
@@ -1314,37 +1317,48 @@ export default class MetamaskController extends EventEmitter {
       }),
     });
 
-    const transactionMetricsRequest = this.getTransactionMetricsRequest();
+    const transactionEventRequest = this.getTransactionEventRequest();
```

Comment:
Is it worth moving this call and the listener creation below into a `_addTransactionControllerListeners`?

Reply:
Moved all the listeners